rfc/src/rfcs/0012-list.md

```diff
@@ -66,17 +66,37 @@ Totals:
 
 A "Standard Harness" is a `#[proof]` harness, while a "Contract Harness" is a `#[proof_for_contract]` harness.
 
+#### Output Schema
```

Comment:
I don't think we should / need to define a formal output schema for the `pretty` output in here. This will likely change as we implement this feature and get feedback from users.